src/store/index.ts

```diff
@@ -1,7 +1,12 @@
-export type StoreType = {
+export type AppWindowSettings = {
   windowWidth: number;
   windowHeight: number;
   windowX: number | undefined;
   windowY: number | undefined;
   windowMaximized?: boolean;
 };
+
+export type DesktopSettings = {
+  basePath?: string;
+  installState?: 'started' | 'installed' | 'upgraded';
```

Comment:
Can you add some documentation explaining what each state represents? Do we have a plan to expand this install state to track more stages during installation so that we can resume at certain state on install retry?